robin_stocks/robinhood/options.py

```diff
@@ -334,43 +334,15 @@ def get_option_market_data_by_id(id, info=None):
 
     """
     instrument = get_option_instrument_data_by_id(id)
-    url = marketdata_options_url()
-    payload = {
-        "instruments" : instrument['url']
-    }
-    data = request_get(url, 'results', payload)
-
-    if not data:
-        data= {
-        'adjusted_mark_price':'',
-        'ask_price':'',
```

Comment:
e.g it makes no sense to check if data['ask_price'] is empty in the caller, and all other dict keys; instead the call should just check if data is None.